### PR TITLE
Add GS_DEFAULT_ACL to Google Cloud Backend

### PR DESCRIPTION
**Pull request by sww314**
Add ability to set the ACL for each Blob/File as it is added. This allows the bucket to be private and the files to be publically readable.

### CODE REVIEW

File: tests/test_gcloud.py [hunk 1]
```diff
@@ -115,6 +115,22 @@ def test_save2(self):
         self.storage._bucket.get_blob().upload_from_file.assert_called_with(
             content, size=len(data), content_type=mimetypes.guess_type(filename)[0])
 
+    def test_save_with_default_acl(self):
+        data = 'This is some test ủⓝï℅ⅆℇ content.'
+        filename = 'ủⓝï℅ⅆℇ.txt'
+        content = ContentFile(data)
+
+        #'projectPrivate', 'bucketOwnerRead', 'bucketOwnerFullControl', 'private', 'authenticatedRead', 'publicRead', 'publicReadWrite'
```
Comment (mikekeda) [inline tests/test_gcloud.py]: Line too long https://www.python.org/dev/peps/pep-0008/#maximum-line-length
Need space before # https://www.python.org/dev/peps/pep-0008/#block-comments

File: tests/test_gcloud.py [hunk 2]
```diff
@@ -115,6 +115,22 @@ def test_save2(self):
...
+        self.storage.default_acl = 'publicRead'
+
+        self.storage.save(filename, content)
+
+        self.storage._client.get_bucket.assert_called_with(self.bucket_name)
+        self.storage._bucket.get_blob().upload_from_file.assert_called_with(
+            content, size=len(data), content_type=mimetypes.guess_type(filename)[0])
+        self.storage._bucket.get_blob().acl.save_predefined.assert_called_with('publicRead')
+
+
```
Comment (mikekeda) [inline tests/test_gcloud.py]: Remove this line - https://www.python.org/dev/peps/pep-0008/#blank-lines

**Comment by edevil** [comment]
This would be great!

**Comment by edevil** [comment]
@sww314 Do you have time to address the comments?

**Comment by sww314** [comment]
@jschneier Can someone merge this pull request? I am using it in production and it works.

**Pull request merged**